### PR TITLE
set a prevent default on enter key when in pantry text box

### PR DESCRIPTION
**Pull request by knpausch**
Summary of the work done:

- Fixes the bug where you go to pantry view and press enter at any point on quantity text box it restarts the whole application (also known as a [application log out](https://social.msdn.microsoft.com/Forums/en-US/d36789e8-969c-4e7d-9083-c02c67822c18/application-logout-whenever-i-hit-enter?forum=aspwebforms))

This branch includes:

- Adds another event listener to inputQuantity element (aka pantry text box)
- Incorporates a conditional to check if 'enter' was pressed w/ key down
- prevents default event of enter key (application log out) 

Contributed to code:

- Kiko

Needs to review:

- Lauren 
- Eleanor

Indicate anything that still needs to be addressed

- You'd think it would it involve our previous conditionals when pressing the 'enter' key however I could tell it was unrelated because I was able to comment them out and the bug still persisted. This was an interesting problem learning about 'enter' key's default function (application log out). Let me know if you have any questions

### CODE REVIEW

File: src/scripts.js [hunk 1]
```diff
@@ -502,4 +503,11 @@ function cookRecipe() {
    removeIngredientsFromPantry().forEach(element => {
         updatePantry(element)
     })
+}
+
+function fixEnter(event){
+    if(event.keyCode === 13){
+        event.preventDefault();
+        addItemToPantry()
+    }
```
Comment (Eleanorgruth) [inline src/scripts.js:512]: Do you think that we should just put this function in the event listener like the searchBar one? Something like this? 
inputQuantity.addEventListener('keydown', (event) => {
  if(event.keyCode === 13) {
        event.preventDefault();
        addItemToPantry()
    }
})

File: src/scripts.js [hunk 2]
```diff
@@ -502,4 +503,11 @@ function cookRecipe() {
    removeIngredientsFromPantry().forEach(element => {
         updatePantry(element)
     })
+}
+
+function fixEnter(event){
+    if(event.keyCode === 13){
```
Comment (Eleanorgruth) [inline src/scripts.js:509]: it also looks like keyCode is deprecated, should we use event.key? https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode

**Review by Eleanorgruth — CHANGES_REQUESTED** [review]

**Review by FrazierLE — APPROVED** [review]

File: src/scripts.js (hunk 1 above)
Comment (knpausch) [reply src/scripts.js]: Ty for this feedback, it is now an implicit function

File: src/scripts.js (hunk 2 above)
Comment (knpausch) [reply src/scripts.js]: All code has been adjusted to use event.key for consistency

File: src/scripts.js [hunk 3]
```diff
@@ -345,7 +351,7 @@ function findRecipeOnClick(event){
 }
 
 function findRecipeOnTab(event){
-    if(event.keyCode === 32 || event.keyCode === 13){
+    if(event.key === " " || event.key === "Enter"){
```
Comment (Eleanorgruth) [inline src/scripts.js:354]: Lol it's just a " ", typical. Thank you!

**Review by Eleanorgruth — APPROVED** [review]

**Pull request merged**